### PR TITLE
Cleanup of sections 1 and 3; moved diagnostics to section 3

### PR DESCRIPTION
**Pull request by fonnesbeck**

### CODE REVIEW

File: notebooks/Section1-Homework.ipynb [hunk 1]
```diff
@@ -0,0 +1,1023 @@
...
+   "metadata": {},
+   "source": [
+    "# Section 1: Homework Exercises\n",
+    "\n",
+    "This material provides some hands-on experience using the methods learned from the first day's material. They focus on building models using real-world data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+<<<<<<< HEAD
```
Comment (twiecki) [inline notebooks/Section1-Homework.ipynb]: left-over merge-conflict

File: notebooks/Section1-Homework.ipynb [hunk 2]
```diff
@@ -0,0 +1,1023 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write your answer here"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+=======
+>>>>>>> e3daaa5fe66bef4e3a6e32e1091bec6a8e8b5216
```
Comment (twiecki) [inline notebooks/Section1-Homework.ipynb]: same

**Comment by twiecki** [comment]
Can you add nbreview?

**Comment by fonnesbeck** [comment]
<p>Added.</p>

File: notebooks/Section1-Homework.ipynb (hunk 1 above)
Comment (fonnesbeck) [reply notebooks/Section1-Homework.ipynb]: Fixed

**Comment by twiecki** [comment]
image

Those three still seem to have invalid json.

**Comment by fonnesbeck** [comment]
Ugh, not sure how that happened. Fixed now.

File: notebooks/Section1_1-Basic_Bayes.ipynb [hunk 3]
```diff
@@ -56,19 +56,15 @@
     "\n",
```
Comment (twiecki) [inline notebooks/Section1_1-Basic_Bayes.ipynb]: <p>Continuous</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fonnesbeck/Bayes_Computing_Course/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

File: notebooks/Section1_3-Model_Comparison.ipynb [hunk 4]
```diff
@@ -30,7 +30,7 @@
     "import matplotlib.pyplot as plt\n",
```
Comment (twiecki) [inline notebooks/Section1_3-Model_Comparison.ipynb]: <p>Didn't even know this changed, probably on arviz side. But maybe best to have the most future-proof code in here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fonnesbeck/Bayes_Computing_Course/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: notebooks/Section1_2-Hierarchical_Models.ipynb [hunk 5]
```diff
@@ -56,7 +56,7 @@
   },
```
Comment (twiecki) [inline notebooks/Section1_2-Hierarchical_Models.ipynb]: <p>Why did you remove the exercise? Even if we skip participants might do them on their own.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fonnesbeck/Bayes_Computing_Course/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='76'/>

File: notebooks/Section1_3-Model_Comparison.ipynb (hunk 4 above)
Comment (fonnesbeck) [reply notebooks/Section1_3-Model_Comparison.ipynb]: <p>Not sure what you are referring to. The warning?</p>

File: notebooks/Section1_2-Hierarchical_Models.ipynb (hunk 5 above)
Comment (fonnesbeck) [reply notebooks/Section1_2-Hierarchical_Models.ipynb]: <p>I'm using it in the homework.</p>

File: notebooks/Section1_3-Model_Comparison.ipynb (hunk 4 above)
Comment (twiecki) [reply notebooks/Section1_3-Model_Comparison.ipynb]: <p>Yeah, exactly.</p>

File: notebooks/Section1_1-Basic_Bayes.ipynb (hunk 3 above)
Comment (fonnesbeck) [reply notebooks/Section1_1-Basic_Bayes.ipynb]: Fixed

**Comment by twiecki** [comment]
:+1:
